src/redux/store.js

```diff
@@ -1,13 +1,19 @@
 import {applyMiddleware, createStore, compose} from 'redux';
 import * as reduxLoop from 'redux-loop';
-
 import middleware from './middleware';
 import reducer from './reducer';
 
-const enhancer = compose(
+let enhancers = [
   applyMiddleware(...middleware),
   reduxLoop.install()
-);
+];
+
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
How about using `if (__DEV__)` here? That's the convention we typically use, and keeping a single way of detecting dev builds makes it easier to write tooling around this in the future.